kustomize/kustomize.go

```diff
@@ -281,9 +282,13 @@ func (km *Manager) Kustomize(namespace string, objects ...runtime.Object) ([]run
 }
 
 func GetUnstructuredObjects(data []byte) ([]runtime.Object, error) {
+	utfData, err := BytesToUtf8Lf(data)
+	if  err  != nil  {
+		return nil, fmt.Errorf("error converting to UTF %s",  err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error converting to UTF %v",  err)
```
err is of type error... and %s represents that a string will be there... So if need to use with %s use err.Error() or better use %v for value

Reply:
Addressed